libs/shared/providers/src/activities/components/ActivityPopover.tsx

```diff
@@ -66,6 +66,7 @@ export const ActivityPopover = ({
         horizontal: 'right',
       }}
       sx={{
+        zIndex: (theme) => theme.zIndex.modal + 2,
```

Comment:
So our notification system does the same thing using `+ 1`. I needed this to be on top of that one so I did `+ 2`. So I'm just following the pattern. However, I agree it would be good to have in the theme.

Reply:
yes, the value itself is ok, just need to be in the oeth theme object for MuiPopover ([here](https://github.com/OriginProtocol/origin-defi/blob/00c2e72bc9dabf3e6bf3ceb9a533e61de323a4b2/libs/oeth/theme/src/theme.tsx#L635))

Reply:
will leave as is for now since it isn't going to matter once we're on the new dapp